assets/js/liturgyOfAnyDay.js

```
@@ -1,134 +1,483 @@
-            break;
-        case 'diocesancalendar':
-            CalendarState.calendarType = 'diocese';
-            CalendarState.calendar = document.querySelector('#calendarSelect').value;
-            break;
-        default:
-            CalendarState.calendarType = '';
-            CalendarState.calendar = '';
-    }
-    getLiturgyOfADay(true);
-});
+const daysInMonth = new Date( now.getFullYear(), now.getMonth() + 1, 0 ).getDate();
```

ESLint

[error] 'daysInMonth' is assigned a value but never used.

(no-unused-vars)